operator/v1/types_network.go

```diff
@@ -507,6 +507,26 @@ type GatewayConfig struct {
 	// The supported values are "Restricted" and "Global".
 	// +optional
 	IPForwarding IPForwardingMode `json:"ipForwarding,omitempty"`
+	// V4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
```

Comment:
Godoc should start with the json tag name, not the Go type name
```suggestion
	// v4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
```